[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-1 (#114270)
openshift/openshift-docs: modules/compliance-increasing-operator-limits.adoc

[id="compliance-increasing-operator-limits_{context}"]
= Increasing Compliance Operator resource limits

[role="_abstract"]

> In some cases, the Compliance Operator might require more memory than the default limits allow. The best way to mitigate this issue is to set custom resource limits.

Reviewer: ```suggestion
In some cases, the Compliance Operator might require more memory than the default limits allow. You can mitigate this issue by setting custom resource limits.
```

[id="compliance-increasing-operator-limits_{context}"]
= Increasing Compliance Operator resource limits

[role="_abstract"]

> In some cases, the Compliance Operator might require more memory than the default limits allow. You can mitigate this issue is to set custom resource limits.

Reviewer: ```suggestion
In some cases, the Compliance Operator might require more memory than the default limits allow. You can mitigate this issue by setting custom resource limits.
```
